.changeset/eleven-meals-remain.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/localized-rich-text-input': major
+'@commercetools-uikit/rich-text-input': major
```

Comment:
Not sure we need to bump a major here, technically nothing changed 🤔

Reply:
yeah good thinking. 
we can mark it as `patch` but is part of the v12 release anyway :)

Reply:
https://github.com/commercetools/ui-kit/pull/1825/commits/f5fe8d457275c5d8b80ef8ba9d5ff002319d13d3